readme.md

```diff
@@ -75,6 +75,7 @@
 - [Custom Native](https://github.com/SirNavith/tailwindcss-custom-native) - Leverages Tailwind CSS's configuration to allow the creation of utilities.
 - [Truncate Multiline](https://github.com/jhta/tailwindcss-truncate-multiline) - Adds utilities to truncate multi-line text elements.
 - [Debug Screens](https://github.com/jorenvanhee/tailwindcss-debug-screens) - Adds a component that shows the currently active screen (responsive breakpoint).
+- [Dark Mode with Class](https://github.com/danestves/tailwindcss-darkmode) - Adds dark variants based on CSS classes.
```

Comment:
Can you surround `dark` with ticks (<code>`</code>)? After that change, it's good to me.

Reply:
Thank you! i don't know why that not put in the markdown, done